yash-datalake/nested/blobTriggerdFunction.json

```diff
@@ -1,389 +1,298 @@
 {
-    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
-    "contentVersion": "1.0.0.0",
-    "parameters": {
-		"kibanaFunctionHost":{
-				
-				"defaultValue": "demotriggerapp1234.azurewebsites.net",
-				"type": "String"
-		},
-		"kibanaFunctionPath":{
-			
-				"defaultValue": "/api/AddKibanaDashboard",
-				"type": "String"
-		},
-		"EsIP":{
-				"defaultValue": "104.209.146.12",
-				"type": "String"
-		},
-        "serverfarms_EastUS2Plan_name": {
-            "defaultValue": "EastUS2Plan",
-            "type": "String"
-        },
-		
-        "sites_demotriggerapp1234_name": {            
-			"defaultValue": "demotriggerapp1234",
-            "type": "String"
-        },
-		"location": {
-		  "type": "string",
-		  "defaultValue": "[resourceGroup().location]",
-		  "metadata": {
-			"description": "Location for all resources."
-		  }
-		},
-		"config_web_name_4": {
-            "defaultValue": "web",
-            "type": "String"
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "location": {
+      "type": "string",
+      "metadata": {
+        "description": "Location for all resources."
+      }
+    },
+    "kibanaFunctionHost": {
+      "type": "String"
+    },
+    "kibanaFunctionPath": {
+      "type": "String"
+    },
+    "EsIP": {
+      "type": "String"
+    },
+    "serverfarms_EastUS2Plan_name": {
+      "type": "String"
+    },
+    "sites_demotriggerapp1234_name": {
+      "type": "String"
+    },
+    "config_web_name_4": {
+      "type": "String"
+    },
+    "storage_name": {
+      "type": "string"
+    },
+    "storageConnectionString": {
+      "type": "String"
+    }
+  },
+  "variables": {
+    "_artifactsLocation": "https://github.com/ajos1993/yash-datalake/raw/master/",

```

Comment:
See [BP guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#deployment-artifacts-nested-templates-scripts) for how to handle artifacts